### PR TITLE
N44: Rerun

### PR DESCRIPTION
**Pull request by Wumpf**
Added `Rerun` to tooling update.

Part of  #1317

### CODE REVIEW

File: content/news/044/index.md [hunk 1]
```diff
@@ -74,6 +74,40 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- Python packages now work with Ubuntu-20.04.
+- Many performance & bug fixes as well as compile time improvements.
+
+There's a growing community on [Discord](https://discord.gg/npTFxYR9)
+that can join if you have questions,
+comments or just want to follow the latest development.
+The [Github project](https://github.com/rerun-io/rerun) is MIT/Apache
+licensed and open to contribute for everyone,
+be it with suggestions, bugs or PRs.
+
```
Comment (AngelOnFira) [inline content/news/044/index.md:110]: If we can move the links to the bottom of the section, that would be great :)

```suggestion
There's a growing community on [Discord][rerun-discord]
that can join if you have questions,
comments or just want to follow the latest development.
The [Github project][rerun-github] is MIT/Apache
licensed and open to contribute for everyone,
be it with suggestions, bugs or PRs.

[rerun-discord]: https://discord.gg/npTFxYR9
[rerun-github]: https://github.com/rerun-io/rerun

```

Comment (Wumpf) [reply content/news/044/index.md]: sure why not

Comment (Wumpf) [reply content/news/044/index.md]: oh, typo in there grammar no work

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks!

**Pull request merged**